server/apps/distributed-app/src/test/java/org/apache/james/CassandraRabbitMQLegacyQuotaJamesServerTest.java

```diff
@@ -22,8 +22,11 @@
 import org.apache.james.jmap.JmapJamesServerContract;
 import org.apache.james.modules.RabbitMQExtension;
 import org.apache.james.modules.TestJMAPServerModule;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
+@Disabled("Disabled to save build time. Legacy quota was a long time ago that we assume people migrated." +
+    "Therefore we do not need to test it on every build.")
```

Comment:
IMO let's simply remove the test

Reply:
removed in https://github.com/apache/james-project/pull/2814